lib/parsers/iso_base_media_file_format/decoder.rb

```diff
@@ -0,0 +1,1039 @@
+# This class provides generic methods for parsing file formats based on QuickTime-style "atoms", such as those seen in
+# the ISO base media file format (ISO/IEC 14496-12), a.k.a MPEG-4, and those that extend it (MP4, CR3, HEIF, etc.).
+#
+# For more information on atoms, see https://developer.apple.com/library/archive/documentation/QuickTime/QTFF/QTFFChap1/qtff1.html
+# or https://b.goeswhere.com/ISO_IEC_14496-12_2015.pdf.
+#
+# The vast majority of the methods have been commented out here. This decision was taken to expedite the release of
```

Comment:
I would mark this with a `# TODO:`, so it's more visible that we (or anyone else) expect to get back to this, sooner or later.